xml/System/Span`1+Enumerator.xml

```diff
@@ -56,9 +55,23 @@ The enumerator does not have exclusive access to the collection; therefore, enum
         <ReturnType>T@</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the item at the current position of the enumerator.</summary>
+        <value>The element in the <see cref="T:System.Span`1" /> at the current position of the enumerator.</value>
+        <remarks>
+      <format type="text/markdown"><![CDATA[  
+
+`Current` is undefined under any of the following conditions: 
+
+- Immediately after the enumertor is created, the enumerator is positioned before the first element in the span. <xref:System.Span%601.Enumerator.MoveNext%2A> must be called to advance the enumerator to the first element of the collection before reading the value of `Current`.
```

Comment:
Is it fine to refer to `Span<T>` as *collection*, as in the last sentence? Also given that in the first sentence it's referred as *the span*.

Reply:
No, it's not, @pkulikov. Thanks for catching it.